exec ${PAGER:-/usr/bin/less} "$0" || exit 1
Executing tests from //compilers/clsmith:clsmith_test
-----------------------------------------------------------------------------
I0308 09:14:50.206932 4437906880 test.py:79] Running pytest with arguments: ['/private/var/tmp/_bazel_cec/ce66aefd86f9b09aced8556389d9936d/sandbox/darwin-sandbox/204/execroot/phd/bazel-out/darwin-fastbuild/bin/compilers/clsmith/clsmith_test.runfiles/phd/compilers/clsmith/clsmith_test.py', '-vv', '--maxfail=1', '--durations=1']
fatal: --local can only be used inside a git repository
============================= test session starts ==============================
platform darwin -- Python 3.6.5, pytest-3.4.1, py-1.5.2, pluggy-0.6.0 -- /usr/local/opt/python/bin/python3.6
cachedir: .pytest_cache
benchmark: 3.1.1 (defaults: timer=time.perf_counter disable_gc=False min_rounds=5 min_time=0.000005 max_time=1.0 calibration_precision=10 warmup=False warmup_iterations=100000)
rootdir: /private/var/tmp/_bazel_cec/ce66aefd86f9b09aced8556389d9936d/sandbox/darwin-sandbox/204/execroot/phd/bazel-out/darwin-fastbuild/bin/compilers/clsmith/clsmith_test.runfiles/phd, inifile:
plugins: benchmark-3.1.1, mock-1.10.0, cov-2.5.1
collecting ... collected 2 items

compilers/clsmith/clsmith_test.py::test_Exec_no_args PASSED              [ 50%]
compilers/clsmith/clsmith_test.py::test_Exec_invalid_argument PASSED     [100%]

=========================== slowest 1 test durations ===========================
18.46s call     compilers/clsmith/clsmith_test.py::test_Exec_no_args
========================== 2 passed in 18.50 seconds ===========================
